packages/embed/e2e/spec/functional/widget-with-parameters.spec.ts

```diff
@@ -0,0 +1,16 @@
+describe('Widget with parameters', () => {
+  before(() => {
+    cy.visit('/widget-with-params-js.html?foo=foo')
```

Comment:
I dont think we need this as a separate file, we can reuse existing `demo-js.html`. I dont want the collection of demos to grow to many files like it did in the old embed (and it was a mess).